src/main/io/dashboard.c

```diff
@@ -71,11 +72,20 @@
 #include "sensors/sensors.h"
 
 
+PG_REGISTER_WITH_RESET_TEMPLATE(dashboardConfig_t, dashboardConfig, PG_DASHBOARD_CONFIG, 0);
+
+PG_RESET_TEMPLATE(dashboardConfig_t, dashboardConfig,
+    .device = DASHBOARD_I2C_INSTANCE,
+    .address = DASHBOARD_I2C_ADDRESS,
+);
+
 #define MICROSECONDS_IN_A_SECOND (1000 * 1000)
 
 #define DISPLAY_UPDATE_FREQUENCY (MICROSECONDS_IN_A_SECOND / 5)
 #define PAGE_CYCLE_FREQUENCY (MICROSECONDS_IN_A_SECOND * 5)
 
+static busDevice_t bus;
```

Comment:
Not sure here, but maybe using `busDevice_t *bus` (or  `busDevice_t * const bus`) is better - it will remove `&` from tons of following lines and code refactoring may be easier in future (passing bus/dashboard as parameter)